case-lib/opt.sh

```diff
@@ -13,48 +13,46 @@ declare -A OPT_NAME OPT_HAS_ARG OPT_VAL OPT_DESC
 # option setup && parse function
 func_opt_parse_option()
 {
-    local _op_temp_script
-    local -A _op_short_lst _op_long_lst
-
     _func_case_dump_description()
     {
          grep '^##' "$SCRIPT_NAME" | sed 's/^## //g'
     }
 
-    # Asks getopt to validate command line input and generate $_op_temp_script
-    #
-    # Also initialize the global _op_short_list and _op_long_list
-    # "reversed maps" (reversed compared to the user's OPT_ maps) that
-    # map valid command line input to the corresponding
-    # one-character option code name. Example:
-    #   _op_short_lst=
-    #        ( [-t]="t" [-d]="d" [-l]="l" [-h]="h" )
-    #   _op_long_lst=
-    #        ( [--help]="h" [--duration]="d" [--tplg]="t" [--loop]="l" )
```

Comment:
I don't see your replacement for this comment as an improvement, you removed a fair bit for no clear reason and you broke the indentation.

Reply:
I think the intend is fixed in `_func_case_dump_description`